projects/core/src/auth/user-auth/services/oauth-lib-wrapper.service.spec.ts

```diff
@@ -119,6 +145,7 @@ describe('OAuthLibWrapperService', () => {
 
       (service as any)['initialize']();
 
+      console.log(winRef);
```

Comment:
Do we need this `console.log` or it's just a leftover?

Reply:
Leftover, removed.